apps/api/src/campaign-application/campaign-application.service.spec.ts

```diff
@@ -1,83 +1,267 @@
 import { Test, TestingModule } from '@nestjs/testing'
 import { CampaignApplicationService } from './campaign-application.service'
 import { CreateCampaignApplicationDto } from './dto/create-campaign-application.dto'
-import { BadRequestException, HttpStatus } from '@nestjs/common'
+import { BadRequestException } from '@nestjs/common'
+import { CampaignApplicationState, CampaignTypeCategory, Person } from '@prisma/client'
+import { prismaMock, MockPrismaService } from '../prisma/prisma-client.mock'
+import { EmailService } from '../email/email.service'
+import { OrganizerService } from '../organizer/organizer.service'
 
 describe('CampaignApplicationService', () => {
   let service: CampaignApplicationService
 
+  const mockPerson = {
+    id: '3ae36b22-2c46-4038-aa17-5427e2c082e5',
+    firstName: 'Martototto',
+    lastName: 'dfsdf',
+    email: 'martbul01@gmail.com',
+    phone: null,
+    createdAt: new Date('2024-06-28T21:13:29.210Z'),
+    updatedAt: new Date('2024-06-28T21:13:29.210Z'),
+    newsletter: true,
+    helpUsImprove: true,
+    address: null,
+    birthday: null,
+    emailConfirmed: false,
+    personalNumber: null,
+    companyId: null,
+    keycloakId: '5a617c6f-4210-4ac3-8e1e-0464ef99f2e5',
+    stripeCustomerId: null,
+    picture: null,
+    profileEnabled: true,
+    company: null,
+    beneficiaries: [],
+    organizer: { id: 'ffdbcc41-85ec-476c-9e59-0662f3b433af' },
+  } as Person
+
+  const mockNewCampaignApplication = {
+    campaignName: 'Test Campaign',
+    organizerName: 'Test Organizer',
+    organizerEmail: 'testemail@gmail.com',
+    organizerPhone: '123456789',
+    beneficiary: 'Test beneficary',
+    organizerBeneficiaryRel: 'Test organizerBeneficiaryRel',
+    goal: 'Test goal',
+    history: 'Test history',
+    amount: '1000',
+    description: 'Test description',
+    campaignGuarantee: 'Test guarantee',
+    otherFinanceSources: 'Test otherFinanceSources',
+    otherNotes: 'Test otherNotes',
+    category: CampaignTypeCategory.medical,
+  }
+
+  const mockCampaigns = [
```

Comment:
Can you create `__mocks__` folder and have one campaignMocks file in it, and export mockCampaigns from it?